packages/styles/scss/components/footer/_legal-nav.scss

```diff
@@ -33,60 +33,91 @@
   }
 
   .#{$prefix}--legal-nav__list {
-    @include carbon--make-col-ready;
-
-    display: flex;
-    flex-wrap: wrap;
-    padding-bottom: $carbon--layout-04;
-  }
-
-  .#{$prefix}--legal-nav__holder {
-    @include carbon--make-col-ready;
-    @include carbon--breakpoint-down('lg') {
-      @include carbon--make-col(4, 4);
+    width: 100%;
+    padding: 0 0 $carbon--layout-04 $carbon--spacing-05;
 
+    &.#{$prefix}--legal-nav__micro {
       display: flex;
-      flex-direction: column;
+      padding-bottom: 0;
       padding-left: 0;
+      flex-direction: column-reverse;
 
       @include carbon--breakpoint('md') {
-        @include carbon--make-col(2, 4);
+        padding-left: 0;
+      }
+
+      @include carbon--breakpoint('lg') {
+        justify-content: space-between;
+        flex-direction: row;
+      }
 
-        display: flex;
-        flex-direction: column;
+      .#{$prefix}--legal-nav__holder {
+        column-count: inherit;
+        margin-left: 0;
+        border-top: 1px solid $ui-03;
+        padding: 0 $carbon--spacing-05;
 
-        &:not(:first-child) {
-          padding-left: $carbon--layout-01;
+        @include carbon--breakpoint('md') {
+          display: flex;
+          align-items: center;
+          padding: 0 $carbon--spacing-05 0;
         }
 
-        &:nth-child(odd):not(:first-child) {
-          padding-left: 0;
+        @include carbon--breakpoint('lg') {
+          border-top: none;
         }
-      }
-    }
 
-    @include carbon--breakpoint('lg') {
-      @include carbon--make-col(1, 4);
+        .#{$prefix}--legal-nav__list-item {
+          padding: 0;
+          margin-right: 0;
 
-      display: flex;
-      flex-direction: column;
+          @include carbon--breakpoint('md') {
+            margin-right: $carbon--spacing-07;
+          }
 
-      &:nth-child(odd) {
-        padding-left: $carbon--spacing-06;
+          a {
+            display: flex;
+            align-items: center;
+            width: 100%;
+            height: $carbon--spacing-09;
+          }
+        }
       }
-      &:first-child {
-        @include carbon--make-col-offset(1, 4);
 
-        padding-left: $carbon--spacing-03;
+      .#{$prefix}--locale-btn__container {
+        margin: 0;
+        text-align: right;
```

Comment:
@kennylam looks like this text-align is causing the locale modal text to be adjust as well

Reply:
@ariellalgilmore Thanks, good catch! That doesn't even need to be there any more.